styles/variables-carbon.less

```diff
@@ -78,7 +78,7 @@
 
 // Slider
 // ---------------------------------------
-@agate-slider-knob-active-transform: @agate-translate-center scale(1);
+@agate-slider-active-knob-transform: @agate-translate-center scale(1);
```

Comment:
Could we omit `scale(1)`? In my opinion, by default, the scale would be `1`. Please fix me if I'm wrong.

Reply:
If we have to use the `scale`, please use scale`(@agate-slider-knob-resting-state-scale)`

Reply:
I removed the scale. it was not necessary